DxDispatch/models/dml_gemm_graph.json

```diff
@@ -0,0 +1,57 @@
+{
+    "$schema": "./_schema.json",
+    
+    "resources": 
+    {
+        "A": { 
+            "initialValuesDataType": "FLOAT32",
+            "initialValues": { "valueCount": 1024, "value": 1 }
+    },
+        "B": {
+            "initialValuesDataType": "FLOAT32",
+            "initialValues": { "valueCount": 1024, "value": 1 }
+    },
+        "output": {
+            "initialValuesDataType": "FLOAT32",
+            "initialValues": { "valueCount": 1024, "value": 1 }
+    }
+    },
+
+    "dispatchables": 
+    {
+        "gemm": 
+        {
+            "type": "DML_OPERATOR_GEMM",
+            "desc": 
+            {
+                "ATensor": { "DataType": "FLOAT32", "Sizes": [1,1,32,32] },
+                "BTensor": { "DataType": "FLOAT32", "Sizes": [1,1,32,32], "Flags": "DML_TENSOR_FLAG_OWNED_BY_DML" },
+                "OutputTensor": { "DataType": "FLOAT32", "Sizes": [1,1,32,32] },
+                "TransA": "DML_MATRIX_TRANSFORM_NONE",
+                "TransB": "DML_MATRIX_TRANSFORM_NONE",
+                "Alpha": 1.0,
+                "Beta": 1.0
+            },
+            "DmlCompileType": "DmlCompileGraph",
```

Comment:
Can you update Guide.md to document this option and what it does? I'm undecided if it would also be useful as a command-line arg in addition to a JSON option, but perhaps it's too narrow/specific to be generally useful.

Reply:
I might hold off on command-line option if it is not needed.  If added JSON then option can conflict with command-line.  Additionally specifying in operator description one op can be declared with graph compile while another one is defined to use the operator compile.